install/operator/pkg/syndesis/configuration/configuration.go

```diff
@@ -152,6 +152,15 @@ type ResourcesWithVolume struct {
 	VolumeCapacity string
 }
 
+type ResourcesWithPersistentVolume struct {
+	Memory             string
+	VolumeCapacity     string
+	VolumeName         string
+	VolumeAccessMode   string
+	VolumeStorageClass string
```

Comment:
Can we get default values for all of them?

Reply:
No, as some are supposed to be blank by default.

VolumeCapacity & VolumeAccessMode are provided with defaults in config.yaml. These are the minimum to provide the default behaviour of specifying a PVC for claiming a default file PV. The others are optional and will enhance the specification for a particular kind of PV. In fact, it is pointless to specify the others if specifying VolumeName as this bypasses the openshift algorithm for determining a PV & simply allocated the named PV if it exists.